workloads/kube-burner/common.sh

```diff
@@ -11,7 +11,12 @@ if [[ ${INDEXING} == "false" ]]; then
   export METADATA_COLLECTION=false
   unset PROM_URL
 else
-  export PROM_TOKEN=$(oc -n openshift-monitoring sa get-token prometheus-k8s)
+  if [[ ${HYPERSHIFT} == "false" ]]; then
+    export PROM_TOKEN=$(oc -n openshift-monitoring sa get-token prometheus-k8s)
+  else
+    export PROM_TOKEN="dummytokenforthanos"
```

Comment:
Can we do a connection check with the token and fail if its not valid - trying to avoid the case where user doesn't change this.

Reply:
> Can we do a connection check with the token and fail if its not valid - trying to avoid the case where user doesn't change this.

Thanos Querier doesn't have a password yet, so literally any string will work. 

> Also why can't the script install grafana-agent on the management cluster as well?

As management cluster needs a different `KUBECONFIG` file, I thought its kinda overdo to switch config and install it on other cluster, btw `airflow` does it by default on all cluster.
I am okay to add another optional input to pass management config file and install it on the fly, lmk.